src/components/TouchCapture/index.js

```diff
@@ -0,0 +1 @@
+export { default as MTouchCapture } from './src/TouchCapture.vue';
```

Comment:
Personal mental note, we should categorize these helper componenta a bit more in the styleguide. Maybe it's time to organize our list beyond alphabetical.